ClemBot.Docs/docs/Tags.md

```diff
@@ -186,32 +186,46 @@ A group of commands used to preview, set, or reset the prefix for tags.
 !tag prefix [<prefix> | reset]
 ```
 
-#### Examples
+#### Tag Prefix
 
-##### View Tag
+Gets the current tag prefix.
 
-```
+##### Example
+
+```txt title="Display the current tag prefix"
 !tag prefix
 ```
 
-##### Set Tag
-
-```
-!tag prefix @
+```txt title="Set a custom tag prefix"
+!tag prefix #
 ```
-:::note
-Custom tag prefixes can have more than one character.
-:::
```

Comment:
why are we removing this?

Reply:
Good catch, I thought I had included an example demonstrating this - I had not.